libr/cons/grep.c

```diff
@@ -329,7 +329,7 @@ R_API int r_cons_grepbuf(char *buf, int len) {
 			}
 			R_FREE (cons->grep.json_path);
 		} else {
-			char *out = r_print_json_indent (buf, I(use_color));
+			char *out = r_print_json_indent (buf, I(use_color), ' ', 2);
```

Comment:
i was thinking in passing a string and just concatenate it instead of specifying a char and the length which can be accomplished too with r_str_pad()